### DISCUSSION TITLE
How to start a network quadlet at system boot?

### DISCUSSION THREAD

**Discussion by ChrisHills463** [containers/podman#24055] [Q&A]
I would like to create podman networks at boot but so far I have been unsuccessful. I have created a quadlet which systemd recognizes and is able to start, but I cannot "enable" it with `systemctl`. The error displayed is `Failed to enable unit: Unit /run/systemd/generator/traefik-network.service is transient or generated.` This is what I have tried so far:-

```bash
sudo tee /etc/containers/systemd/traefik.network >/dev/null <<EOF
[Unit]
Description=Podman network traefik

[Network]
Driver=bridge
NetworkName=traefik

[Install]
WantedBy=default.target
EOF

sudo systemctl daemon-reload

sudo systemctl start traefik-network.service

sudo systemctl enable traefik-network.service 
# Failed to enable unit: Unit /run/systemd/generator/traefik-network.service is transient or generated.
```

I would appreciate any help!

**Comment by Luap99** [comment]
https://docs.podman.io/en/latest/markdown/podman-systemd.unit.5.html#enabling-unit-files

In general there should not be a need to enable the network units as the container unit should pull it in its dependencies automatically

**Reply by ChrisHills463** [reply]
Will this work for networks defined as `external` in the container?

**Reply by Luap99** [reply]
I don't understand this question, what do you mean by external?

**Reply by ChrisHills463** [reply]
In `podman-compose.yml`:-

```
networks:
  traefik:
    external: true
```

This is because I have a collection of containers that need to communicate but are not part of the same application, and I need to be able to manage each container independently (e.g. restarting/upgrading without affecting the others).
